Lib/test/_test_multiprocessing.py

```diff
@@ -6844,6 +6844,28 @@ def f(x): return x*x
         self.assertEqual("332833500", out.decode('utf-8').strip())
         self.assertFalse(err, msg=err.decode('utf-8'))
 
+    @support.requires_fork()
+    def test_forked_not_started(self):
+        # gh-134381
+        # Ensure that a forked parent process, which has not been started yet,
+        # can be initiated within the child process.
+
+        ctx = multiprocessing.get_context("fork")   # local “fork” cont
+        q = ctx.Queue()
```

Comment:
The multiprocessing tests are structured a bit strangely, so to avoid running a fork-specific test in the spawn test suite, we should use something like:

```suggestion
        if multiprocessing.get_start_method() != "fork":
            self.skipTest("fork specific test")

        q = multiprocessing.Queue()
```

Reply:
It seems reasonable, but could this be a deprecated test code in the future, since version 3.14 or later sets the default to spawn?

Reply:
This is how the other tests within this file are structured. Fork is not deprecated.

Reply:
Thanks, noted.